src/transformers/models/vmamba/modeling_vmamba.py

```diff
@@ -0,0 +1,711 @@
+# coding=utf-8
+# Copyright 2022 Derk Mus The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch VMamba model. """
+
+
+import math
+import os
+from functools import partial
+from typing import Callable
+
+import torch
+import torch.nn.functional as F
+import torch.utils.checkpoint as checkpoint
+from mamba_ssm.ops.selective_scan_interface import selective_scan_ref  # selective_scan_fn
```

Comment:
I think this dependency should be removed

Reply:
Yep!

Reply:
done